packages/shared/components/organisms/ReceiveAddressSection.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="typescript">
+    import { appSettings } from '@core/app'
+    import { QR, Text } from 'shared/components'
+    import { AddressBox, OnboardingButton } from 'shared/components/atoms'
+    import { localize } from '@core/i18n'
+    import { selectedAccount } from '@core/account'
+    import { FontWeightText } from 'shared/components/Text.svelte'
+    import { openPopup } from '@lib/popup'
+
+    export let addressFontSize = 'base'
+
+    let addressBoxElement: AddressBox
+
+    $: recieveAddress = $selectedAccount.meta.publicAddresses[0].address
+    $: darkModeEnabled = $appSettings.darkMode
+
+    function handleSendClick() {
+        openPopup({
+            type: 'sendForm',
+            overflow: true,
+        })
+    }
+
+    function handleRecieveClick() {
+        addressBoxElement.copyAddress()
+    }
+</script>
+
+<receive-details class="flex flex-col space-y-6 justify-between cursor-pointer">
+    <OnboardingButton primaryText={'Send funds'} secondaryText="Send tokens to an address" onClick={handleSendClick} />
+    <div
+        class="flex flex-col px-4 py-4 space-y-6 rounded-xl"
```

Comment:
```suggestion
        class="flex flex-col px-4 py-4 space-y-6 rounded-xl cursor-pointer"
```
Add the cursor-pointer to this div so that the cursor appears over the QR div. Send button doesn't need the explicit style since it is already added within OnboardingButton